packages/typeorm/src/typeorm.ts

```diff
@@ -1,60 +1,105 @@
 import { Logger as LoggerInterface } from 'typeorm';
-import { Container, ContainerAware, Logger, Plugin } from '@debugr/core';
-import { SqlFormatter } from '@debugr/sql-formatter';
+import { Logger, Plugin, LogLevel, TContextBase, TContextShape } from '@debugr/core';
+import { SqlLogEntry } from './types';
 
-const levelMap = {
-  log: Logger.INFO,
-  info: Logger.INFO,
-  warn: Logger.WARNING,
+const defaultLevelMap = {
+  log: LogLevel.INFO,
+  info: LogLevel.INFO,
+  warn: LogLevel.WARNING,
 };
 
-export class TypeormLogger implements ContainerAware, Plugin, LoggerInterface {
-  readonly id: string = 'typeorm';
+export type TypeORMLoggerOptions = {
+  levelMap?: {
+    log: LogLevel | number;
+    info: LogLevel | number;
+    warn: LogLevel | number;
+  };
+  levelMigration?: LogLevel | number;
+  levelQuery?: LogLevel | number;
+  levelError?: LogLevel | number;
+  levelQuerySlow?: LogLevel | number;
+  levelSchemaBuild?: LogLevel | number;
+};
```

Comment:
`levelError` bych přesunul do `levelMap.error` a ty ostatní bych asi spíš pojmenoval obráceně (`*Level` vs. `level*`), tohle je divný, ale určitě je dobře že jdou nastavit

Reply:
oka